CONTRIBUTING.adoc

```diff
@@ -0,0 +1,93 @@
+= Contributing to Plugin Health Scoring
+
+This page provides information about contributing to Plugin Health Scoring code base.
+
+== Getting started
+
+The project is using Maven with Java 17 and Yarn to be built.
+The entire build process can be handled by Maven.
+Yarn build will bundle the front-end files only.
+
+. Clone the repository
+. Install tools
+.. Java 17.
+.. Maven 3.8.6.
+Optionally, you can use the wrapper in the repository.
+In that case, replace the `mvn` command by `./mvnw` in the command listed bellow.
+.. Node 18.10.0 and Yarn 3.2.4
+. Open the project in your favorite IDE
+
+== Building
+
+If you simply want the project binary, run:
+
+[source,bash]
+----
+mvn package -Dmaven.test.skip
+----
+
+=== Running locally
+
+There are two ways to run the project locally.
+Either way, the application requires some environment variables to be configured.
+
+[source,bash]
+----
+include::.env.example[]
+----
+
+==== CLI
+
+This method is preferred if you intend to submit any modifications to the project.
+
+In one terminal, run:
+
+[source,bash]
+----
+yarn dev
+----
+
+and in a second terminal run:
+
+[source,bash]
+----
+mvn spring-boot:run -Dskip.yarn
+----
+
+==== Docker Compose
+
+If you want to simply run the project locally, you can run:
+
+[source,bash]
+----
+mvn package -Dmaven.test.skip
+docker compose up
+----
+
+NOTE: the environment variable required must be set in a file named `.env`.
+
+== Testing change
+
+The project has unit and integration tests.
+You can run all those with
+
+[source,bash]
+----
+mvn verify
+----
+
+WARNING: The integration tests requires Docker to be installed.
+
+== Proposing changes
+
+All proposed changes are submitted and reviewed through a GitHub pull request.
+To submit a pull request:
+
+. Create a branch prefixed with `feature/` for a new enhancement, `fix/` for a bugfix.
+. Commit your changes to this new branch.
+. Push the branch to your fork of the repository.
+. In GitHub Web UI, click the _New pull request_ button.
```

Comment:
```suggestion
. In GitHub Web UI, click the _New pull request_ button.
```
. In the GitHub Web UI, select the *New pull request* option.

Small suggestion, as we are trying to use select as opposed to click. Again, if this does not make sense, it does not need to be changed. Also, if the button is labeled as New pull request, it can be bolded so that users know what to look for.